lib/router/tasks.js

```diff
@@ -0,0 +1,62 @@
+const Case = require('@ukhomeoffice/taskflow/lib/models/case');
+const { Router } = require('express');
+const router = Router({ mergeParams: true });
+
+const roleQueryBuilders = {
+  applicant: require('../query-builders/applicant'),
+  ntco: require('../query-builders/ntco')
+};
```

Comment:
Not sure I see the benefit of creating an object to contain 2 required files - the pattern we use elsewhere is to have an index.js in the root of the query-builder directory and destructure the required modules

Reply:
a la
```js
const { applicant, ntco } = require('../query-builders');
```

Reply:
This is missing the other ones that exist - I can add an index to the queryBuilders dir instead?

Reply:
yeah it's more just a comment, not a blocker, we use an index.js file in most other places like this, IMO it would be nicer, but I also don't actually care that much

Reply:
better?

Reply:
🤗